src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/AdeAggregatesVerifier.java

```diff
@@ -0,0 +1,72 @@
+package it.gov.pagopa.rtd.ms.rtdmsdecrypter.service;
+
+import com.opencsv.bean.BeanVerifier;
+import com.opencsv.exceptions.CsvConstraintViolationException;
+import it.gov.pagopa.rtd.ms.rtdmsdecrypter.model.AdeTransactionsAggregate;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+
+/**
+ * Implementation of {@link BeanVerifier}, used to verify the validity of the
+ * {@link AdeTransactionsAggregate} records extracted from the decrypted file.
+ */
+public class AdeAggregatesVerifier implements BeanVerifier<AdeTransactionsAggregate> {
+
+  private static final ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+
+  private static final Validator validator = factory.getValidator();
```

Comment:
What's the point in having multiple static instances of factory and validator around the codebase? Do we foresee the need of different kind of validator or any future customization? 
If not, maybe it is better to place the common instantiation in a @Configuration class and inject the validator where needed. How is it sound?